lib/perl/Genome/WorkflowBuilder/Converge.pm

```diff
@@ -73,4 +74,37 @@ sub notify_output_link {
     return;
 }
 
+sub get_ptero_builder_task {
+    require Ptero::Builder::Detail::Workflow::Task;
```

Comment:
It seems odd that you'd have to read through every subroutine implementation in a module to discover its potential dependencies. Maybe all the `requires` should be at the top of the module? Or, if its fine to read through every implementation to find dependencies, why not lean on the autoloader and omit the `require` entirely? Are we trying to avoid a compile time dependency on `Ptero::Builder::Detail::Workflow::Task`?